src/sentry/deletions/defaults/group.py

```diff
@@ -1,6 +1,59 @@
 from __future__ import absolute_import, print_function
 
-from ..base import ModelDeletionTask, ModelRelation
+from sentry import eventstore, nodestore
+from sentry.models import Event
+
+from ..base import BaseDeletionTask, BaseRelation, ModelDeletionTask, ModelRelation
+
+
+class GroupNodeDeletionTask(BaseDeletionTask):
```

Comment:
Will be expanded to handle EventAttachment in future

Reply:
We might also want to handle UserReports this way, since we don't always write a group_id when these are received